R/has-name.R

```diff
@@ -13,6 +13,5 @@
 #' has_name(iris, "Species")
 #' has_name(mtcars, "gears")
 #' @export
-has_name <- function(x, name) {
-  name %in% names2(x)
-}
+#' @name has_name
+has_name <- rlang::has_name
```

Comment:
oh I see the new roxygen version does not move reexports to a common file now, so we can do that. It does add a link to rlang which I don't think would be useful though:

```
\keyword{internal}
\description{
These objects are imported from other packages. Follow the links
below to see their documentation.

\describe{
  \item{rlang}{\code{\link[rlang]{has_name}}}
}}
```